modules/nodes-edge-remote-workers-strategies.adoc

```diff
@@ -148,13 +146,13 @@ tolerations:
 - key: "node.kubernetes.io/not-ready"
   operator: "Exists"
   effect: "NoExecute" <2>
-  tolerationSeconds: 600
+  tolerationSeconds: "300" <3>
```

Comment:
probably better to keep the old value or have a different one, because 300s (5m) is the default behaviour for this particular toleration

Reply:
Is the default 5 minutes or no limit? Or, is it that if the `tolerationSeconds` variable is not applied then the default is no limit?

Reply:
if there is no toleration entry with one of these two keys, then a new entry will be added with 5 minutes long `tolerationSeconds` for each missing key